container/single-instance-codespaces/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# SPDX-License-Identifier: MIT
+#!BuildTag: openqa-single-instance-codespaces:latest opensuse/openqa-single-instance-codespaces:latest opensuse/openqa-single-instance-codespaces:%PKG_VERSION% opensuse/openqa-single-instance-codespaces:%PKG_VERSION%.%RELEASE%
+
+# hadolint ignore=DL3006
+FROM openqa-single-instance
+
+# labelprefix=org.opensuse.openqa-single-instance-codespaces
+LABEL org.opencontainers.image.title="openQA single-instance container for Codespaces"
+LABEL org.opencontainers.image.description="A complete openQA instance composed of all necessary components to execute openQA tests including an openQA worker"
+LABEL org.opencontainers.image.version="%PKG_VERSION%.%RELEASE%"
+LABEL org.opensuse.reference="registry.opensuse.org/opensuse/openqa-single-instance-codespaces:%PKG_VERSION%.%RELEASE%"
+LABEL org.openbuildservice.disturl="%DISTURL%"
+LABEL org.opencontainers.image.created="%BUILDTIME%"
+# endlabelprefix
+
+# install Codespaces requirements
+# hadolint ignore=DL3037
+RUN zypper in -y awk tar && \
```

Comment:
Why not add those to the existing container? Seems like they would be useful anyway and we wouldn't need yet another container to maintain - and I wonder where to test this? openQA-in-openQA tests or somewhere else?

Reply:
See https://github.com/os-autoinst/openQA/pull/5818#issuecomment-2269059690

It violated *AC2* as discussed during the estimation (yes, I would also think adding just those basic tools wouldn't hurt, but the ticket says otherwise).

I do not know if codespaces containers can be tested automatically, for manual testing I guess just start the environment should do the trick (although it is unclear to me if I can do that without changing the devcontainer definition).

Reply:
testing codespaces is a bit cumbersome. you would need to branch the obs project (and enable publishing) and then in your github fork point to the container from your fork.

Reply:
regarding awk and tar - The downside of having two containers is that it makes it even harder to test any changes, so I personally would vote against AC2.